connexion/decorators/security.py

```diff
@@ -71,7 +71,10 @@ def wrapper(request):
                     token_response=token_request
                 )
             token_info = token_request.json()  # type: dict
-            user_scopes = set(token_info['scope'])
+            if isinstance(token_info['scope'], list):
+                user_scopes = set(token_info['scope'])
+            else:
+                user_scopes = set(token_info['scope'].split())
```

Comment:
No, I'm splitting by whitespace. As I mentioned in #475, the spec says the scope items are space-separated.